src/openrct2/object/ObjectManager.cpp

```diff
@@ -538,7 +538,7 @@ class ObjectManager final : public IObjectManager
                 if (entry.HasValue())
                 {
                     const auto* ori = _objectRepository.FindObject(entry);
-                    if (ori == nullptr && entry.GetType() != ObjectType::ScenarioText)
+                    if (ori == nullptr)
```

Comment:
I would leave this as-is. A missing STEX should not stop a save from loading.

Reply:
Fair. At the same time, we'll be distributing all known scenario text objects. There would be something wrong with the OpenRCT2 install if the objects cannot be found.